src/context.rs

```diff
@@ -50,11 +51,13 @@ impl<'a> Context<'a> {
                 })
             });
 
-        Context::new_with_dir(arguments, path)
+        let logical_path: Option<PathBuf> = arguments.value_of("logical-path").map(From::from);
+
+        Context::new_with_dir(arguments, path, logical_path)
     }
 
     /// Create a new instance of Context for the provided directory
```

Comment:
**nitpick:**
Could the comment please be updated to reflect the new argument. Perhaps explaining when `logical_path` is to be used.

Reply:
Just pushed a new commit for this